app/lib/backend/storage_rewrite.rb

```diff
@@ -25,7 +25,6 @@ def rewrite_all
 
       Cinstance.find_each do |cinstance|
         cinstance.send(:update_backend_application)
-        cinstance.send(:update_backend_user_key_to_application_id_mapping)
```

Comment:
We should not remove this or is it replaced by `update_backend_application`?

Reply:
@hallelujah What this does is: if the user_key changed, remove the whole application_plan, and create it again with the new user_key. We do not need that anymore.
`update_backend_application` updates all attributes, and it does not contain user_key...  and afaik, we shouldn't add it there. However, we should call `update_provider_backend_service_if_user_key_changed` instead imo.

Reply:
I think you meant "remove the whole application", not "application_plan".

Reply:
right hahahahaha 🙈 😂